app/Http/Controllers/User/userRegisterController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Hash;
+
+class userRegisterController extends Controller
+{
+    public function showUser(){
+        return view('user.userRegister');
+    }
+    
+    public function userDb(Request $request){
+        
+        //値取得
+        $user_name = $request->input('user_name');
+        $password = $request->input('password');
+        $admin = $request->input('admin');
+        if($admin == null){
```

Comment:
```suggestion
        $admin = $admin == null ? 0 : 1;
```
ここは三項演算子で一行にまとめられます。